autofs-5.1.3 - document ghost option in auto.master man page

From: Ian Kent <raven@themaw.net>

The --ghost pseudo option has been depricated for quite a while now
in favour of the browse option as descibed in auto.master(5).

However it isn't noted in the description of the browse option that
the --ghost option is the same as the browse option.

Signed-off-by: Ian Kent <raven@themaw.net>
---
 CHANGELOG            |    1 +
 man/auto.master.5.in |    3 ++-
 2 files changed, 3 insertions(+), 1 deletion(-)

--- autofs-5.0.7.orig/CHANGELOG
+++ autofs-5.0.7/CHANGELOG
@@ -322,6 +322,7 @@
 - add mount_verbose configuration option.
 - optionally log mount requestor process info.
 - log mount call arguments if mount_verbose is set.
+- document ghost option in auto.master man page.
 
 25/07/2012 autofs-5.0.7
 =======================
--- autofs-5.0.7.orig/man/auto.master.5.in
+++ autofs-5.0.7/man/auto.master.5.in
@@ -176,7 +176,8 @@ can cause performance problem if the ind
 be used with caution. The internal program default is to enable browse
 mode for indirect mounts but the default installed configuration overrides
 this by setting BROWSE_MODE to "no" because of the potential performance
-problem.
+problem. This option does the same as the depricated --ghost option, the
+browse option is preferred because it is used by other autofs implementations.
 .TP
 .I "nobind"
 This is an autofs specific option that is a pseudo mount option and
